src/models/favorite.js

```diff
@@ -0,0 +1,10 @@
+const mongoose = require('mongoose');
+
+const favoriteSchema = new mongoose.Schema({
+  user: mongoose.Types.ObjectId,
+  house: mongoose.Types.ObjectId,
```

Comment:
I made some changes in my code to add ref in the schema. It is necessary to make proper relations.

```suggestion
  user: { type: Schema.Types.ObjectId, ref: 'User' },
  house: { type: Schema.Types.ObjectId, ref: 'House' }
```